src/libraries/System.Text.Json/tests/Serialization/NumberHandlingTests.cs

```diff
@@ -400,6 +400,7 @@ private static void AssertDictionaryElements_StringValues(string serialized)
         }
 
         [Fact]
+        [ActiveIssue("", typeof(PlatformDetection), nameof(PlatformDetection.IsMonoInterpreter))]
```

Comment:
Does this feature break mono interpreter?

Reply:
No, just fails as a long running test. Will either try to make it take a shorter time without losing coverage, or file an issue similar to https://github.com/dotnet/runtime/issues/35927.